app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DummyDatabaseService.kt

```diff
@@ -184,7 +184,7 @@ class DummyDatabaseService : DatabaseService {
         if (!datasets.contains(dataset)) {
             throw IllegalArgumentException("The underlying database does not contain the dataset ${dataset.name}")
         }
-        val newDs = DummyDataset(dataset.id as String, newName, dataset.categories)
+        val newDs = DummyDataset(newName as String, newName, dataset.categories)
```

Comment:
This contradicts the relational model.

Reply:
Right sorry, this was for testing purpose and should not have made it into the commit...